source/Octopus.Server.Client/Repositories/Async/BasicRepository.cs

```diff
@@ -253,6 +254,26 @@ public async Task<List<TResource>> GetAll(CancellationToken cancellationToken)
             return await Client.Get<List<TResource>>(link, parameters, cancellationToken).ConfigureAwait(false);
         }
 
+        public async Task<List<TResource>> FindByPartialName(string partialName, string path, object pathParameters, CancellationToken cancellationToken)
+        {
+            await ThrowIfServerVersionIsNotCompatible(cancellationToken);
```

Comment:
We need to keep the `.ConfigureAwait(false)` as this is library code and it could potentially affect consumers using custom synchronisation contexts:

```suggestion
            await ThrowIfServerVersionIsNotCompatible(cancellationToken).ConfigureAwait(false);
```

Reply:
Huh interesting TIL, Updated it :)